modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -926,8 +926,11 @@ class InternalFFMpegRegister
         std::unique_lock<cv::Mutex> lock(_mutex, std::defer_lock);
         if(!threadSafe)
             lock.lock();
+        if(av_log_initialized)
+            return;
         static InternalFFMpegRegister instance;
         initLogger_();  // update logger setup unconditionally (GStreamer's libav plugin may override these settings)
```

Comment:
Could we move call `initLogger_()` to the constructor of `InternalFFMpegRegister` instead?
C++11 guarantees proper call once for that (static variable).

Reply:
Indeed, that is much cleaner.